File: 002/solve.go
package main

import "fmt"

func main() {
	sum := 0
	fibseq := []int{1, 1, 2}
	for fibseq[2] < 4000000 {
		if fibseq[2] % 2 == 0 {
			sum += fibseq[2]
		}
		fibseq[0] = fibseq[1]
		fibseq[1] = fibseq[2]
		fibseq[2] = fibseq[0] + fibseq[1]
	}
	fmt.Printf("%d\n", sum)
}

File: 002/solve.js
var temp, sum = 0, a = 0, b = 1

while (b < 4e6) {
  temp = a;
  a = b;
  b += temp;
  if ((b & 1) === 0) {
    sum += b;
  }
}

console.log(sum);

